packages/core/core/src/index.ts

```diff
@@ -68,6 +74,10 @@ export interface EmbarkEvents {
   ): void;
 }
 
+export interface ClientConfig {
+  miningMode?: "dev" | "auto" | "always" | "off";
```

Comment:
Could make this an enum (probably doesn't provide any benefit other than users could potentially rely on it in the future)

Reply:
I actually added this for clarity's sake, as it's not being used in these changes. If we need to reuse it in the future, we can make it an enum.

Reply:
👍